tyssue/geometry/base_geometry.py

```diff
@@ -37,6 +37,12 @@ def update_dcoords(sheet):
         else:
             update_periodic_dcoords(sheet)
 
+    @staticmethod
```

Comment:
can you also add a call to the method in update_all(), and remove the call in factory.py?